### PR TITLE
AWS Install Updates

### PR DESCRIPTION
**Pull request by ianf77**
Updated, largely putting iam policies into a table

### CODE REVIEW

**Comment by ianf77** [comment]
I've put the iams policies into a table, it's more convenient than a list two pages long.

**Review by zkayyali812 — APPROVED** [review]

File: stories/topics/proc-aap-aws-deploy-existing-vpc.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[id="proc-aap-aws-deploy-existing-vpc"]
+
+= Deploying an application with an existing VPC
+
+This following procedure uses an existing VPC network to deploy an application.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-deploy-existing-vpc.adoc:5]:
```suggestion
The following procedure uses an existing VPC network to deploy an application.
```

File: stories/topics/proc-aap-aws-deploy-existing-vpc.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.. In the *EC2 KeyPair* field, select your previously created EC2 keypair.
+.. In the *Select VPC* field, select *Existing*.
+.. Ignore all the parameters for *New network configuration*.
+.. In the *For Existing network configuration* field, enter the
+** *Existing VPC ID* 
+** *Existing VPC CIDR Range* 
+** *Existing Private Subnet 1 ID*
+** *Existing Private Subnet 2 ID*
+. Click btn:[Next] to move to Step 3.
+. For Step 3, *Configure stack options* no changes are necessary.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-deploy-existing-vpc.adoc:27]:
```suggestion
. For Step 3, *Configure stack options*, no changes are necessary. 
```

File: stories/topics/ref-aap-aws-iam-policies.adoc [hunk 3]
```diff
@@ -8,25 +8,55 @@ Your AWS account must also be licensed to deploy {AAPonAWS}.
...
-** AmazonRDSReadOnlyAccess
-** AmazonElasticFileSystemReadOnlyAccess
-** AmazonEC2FullAccess
+The application has two deployment options:
+
+. Deployment with new VPC
+. Deployment with existing VPC
+
+
+The following table contains a list of necessary IAM policies
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-aws-iam-policies.adoc:17]:
```suggestion
The following table contains a list of necessary IAM policies:
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Asciidoc artefact and a few other minor comments

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**